configure.ac

```diff
@@ -43,8 +43,8 @@ AM_CONDITIONAL([LINUX], [test "x$linux" = "x1"])
 AM_CONDITIONAL([FREEBSD], [test "x$freebsd" = "x1"])
 
 base_c_warn_flags="-Wall -Wundef -Wpointer-arith"
-debug_c_warn_flags="-Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-missing-field-initializers"
-debug_c_other_flags="-fstack-protector-strong"
+debug_c_warn_flags="-Wextra -Wno-unused-parameter -Wno-sign-compare"
+debug_c_other_flags="-fstack-protector-strong -Wno-missing-field-initializers"
```

Comment:
Maybe the real solution is to test all the warn flags the same as is done for the other flags. Just let me know what is preferred and I'll update.

Reply:
I'm okay with testing all the flags.  That should avoid this problem in the future.

Reply:
OK, updated to test all the debug flags.